public_html/includes/js/develop/createForm_form.js

```diff
@@ -1531,8 +1531,9 @@ function addMetadataStandard(val){
 	}
 
 	var options    = metadataSchema;
-	var identifier = val.split(' : ')[1];
-	var standard   = val.split(' : ')[0];
+	val = val.replace(/\s+/g, '');
```

Comment:
Okay.  I was just checking since you removed the variables identifier and standard that they weren't being given any other information anywhere else or being used anywhere else.  The reason is that in JavaScript if a variable hasn't been declared using the var keyword and is used it takes on global scope which can cause bugs later.  

Ex: ----------------------------------------------------------
function foobar(){
    var foo = bar; // private scoped and limited to function
}
function foobar(){
     foo = bar; // global scope not limited to function
}